Aspire Budgeting/Views/TabBarView/TabBarItemView.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  TabBarItemView.swift
+//  Aspire Budgeting
+//
+
+import SwiftUI
+
+struct TabBarItem {
+  let imageName: String
+  let title: String
+}
+
+struct TabBarItemView: View {
+  let tabBarItem: TabBarItem
+  
+  let selectedIndex: Int
+  let tabBarIndex: Int
+  
+  let defaultColor: Color
+  let selectedColor: Color
+  
+  let font : Font
+  
+  private var displayColor: Color {
+    selected ? selectedColor : defaultColor
+  }
+  
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/845606672c945b4a26becac88e482112dc55b51f/Aspire Budgeting/Views/TabBarView/TabBarItemView.swift#L27"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>TabBarItemView.swift:27</code></td>
    </tr>
  </tbody>
</table>